app/templates/board.html

```diff
@@ -2,35 +2,68 @@
     <div style="text-align: center;">
         <button ng-click="ShowByAssignee = !ShowByAssignee" class = "btn btn-success" ng-hide="ShowByAssignee"> Currently Showing Columns By Attribute 'Status'. <br> Click To Show By 'Assignee'.</button>
         <button ng-click="ShowByAssignee = !ShowByAssignee" class = "btn btn-success" ng-show="ShowByAssignee">Currently Showing Columns By Attribute 'Assignee'. <br> Click To Show By 'Status'.</button>
+        
+        <label class="switch">
+            <input type="checkbox" checked ng-click="ColumnsScroll = !ColumnsScroll">
+            <div class="slider"></div>
+            <div class="text"></div>
+        </label>
     </div>
     <br>
     <div class="col-sm-{{ (12/board_sections.length) | number:0 }} bz-column" ng-repeat="board in board_sections" ng-hide="ShowByAssignee">
         <button ng-click="showDiv = !showDiv" class = "btn btn-primary">{{ board }}</button>
         <div class="list-group" ng-hide="showDiv">
-            <a href="show_bug.cgi?id={{ item.id }}" class="list-group-item" ng-repeat="item in bug_list | filter: { status: board }">
-                <p class="list-group-item-text bz-item-desc">{{ item.desc }}</p>
-                <span class="label label-default">{{ item.status }}</span>
-                <span class="label label-primary" style="background-color:{{ item.product_color }}">{{ item.product }}</span>
-                <span class="label label-info">{{ item.component }}</span>
-                <span class="label label-default">{{ item.assignee }}</span>
-                <small>{{ item.changed }}</small>
-                <span class="bz-item-id">#{{ item.id }}</span>
-            </a>
+            <div ng-hide="ColumnsScroll" style="height:200px; solid #ccc;font:16px/26px Georgia, Garamond, Serif;overflow:auto;">
+                <a href="show_bug.cgi?id={{ item.id }}" class="list-group-item" ng-repeat="item in bug_list | filter: { status: board }">
+                    <p class="list-group-item-text bz-item-desc">{{ item.desc }}</p>
+                    <span class="label label-default">{{ item.status }}</span>
+                    <span class="label label-primary" style="background-color:{{ item.product_color }}">{{ item.product }}</span>
+                    <span class="label label-info">{{ item.component }}</span>
+                    <span class="label label-default">{{ item.assignee }}</span>
+                    <small>{{ item.changed }}</small>
+                    <span class="bz-item-id">#{{ item.id }}</span>
+                </a>
+            </div>
+            <div ng-show="ColumnsScroll" >
```

Comment:
Instead of this duplicate section, if we have the switch button add and remove a class, e.g.
```
.scroll {
  width:400px;
  overflow:auto;
}
```
to the `list-group` above.